web/src/views/home/sections/technology_section.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Container } from 'reactstrap';
+
+const TechnologySection = () => (
+  <section className="section mt-4">
+    <Container>
+      <h2 className="mt-6">Technology</h2>
+      <p className="lead text-justify">
+        Our sleep stage classification algorithm is trained on the Physionet Sleep-EDF Extended dataset which provides
+        153 nights of polysomnographic recordings. The used machine learning approach achieves a 87% accuracy when
```

Comment:
```suggestion
        153 nights of polysomnographic recordings. The use learning based approach achieves a 87% accuracy when
```

Reply:
`The use of a machine learning based approach` ?